var/spack/repos/builtin/packages/rocblas/package.py

```diff
@@ -259,6 +261,7 @@ def cmake_args(self):
             self.define("BUILD_CLIENTS_BENCHMARKS", "OFF"),
             self.define("BUILD_CLIENTS_SAMPLES", "OFF"),
             self.define("RUN_HEADER_TESTING", "OFF"),
+            self.define("ROCM_OPENMP_EXTRAS_DIR", self.spec["rocm-openmp-extras"].prefix),
```

Comment:
If you want to use that dependency in `cmake_args()`, you'll need to make it at least a `build` (not just `test`) dependency.

Reply:
Moved ROCM_OPENMP_EXTRAS_DIR to run_tests since rocm-openmp-extras is added only for test.

Reply:
Restricted ROCM_OPENMP_EXTRAS_DIR to test